Documentation/Getting-Started/glossary.md

```diff
@@ -0,0 +1,130 @@
+
+### OpenShift
+
+[OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform) Container Platform is a distribution of the Kubernetes container platform.
+
+### PVC Storage Cluster
+
+In a [PersistentVolumeClaim-based cluster](../CRDs/Cluster/pvc-cluster.md), the Ceph persistent data is stored on volumes requested from a storage class of your choice.
+
+### Stretch Storage Cluster
+
+A stretched cluster is a deployment model in which two or more virtualization host servers are part of the same logical cluster but are located in separate geographical locations. To support this scenario, Ceph has integrated support for [stretch clusters](../CRDs/Cluster/stretch-cluster.md).
+
+### Toolbox
+
+The [Rook toolbox](../Troubleshooting/ceph-toolbox.md) is a container with common tools used for rook debugging and testing.
+
+## Ceph
+
+[Ceph](https://docs.ceph.com/en/latest/) is a distributed network storage and file system with distributed metadata management and POSIX semantics. See also the [Ceph Glossary](https://docs.ceph.com/en/latest/glossary/). Here are a few of the important terms to understand:
+
+* [Ceph Monitor](https://docs.ceph.com/en/latest/glossary/#term-Ceph-Monitor) (MON)
+* [Ceph Manager](https://docs.ceph.com/en/latest/glossary/#term-Ceph-Manager) (MGR)
+* [Ceph Metadata Server](https://docs.ceph.com/en/latest/glossary/#term-MDS) (MDS)
+* [Object Storage Device](https://docs.ceph.com/en/latest/glossary/#term-OSD) (OSD)
+* [RADOS Block Device](https://docs.ceph.com/en/latest/glossary/#term-Ceph-Block-Device) (RBD)
+* [Ceph Object Gateway](https://docs.ceph.com/en/latest/glossary/#term-Ceph-Object-Gateway) (RGW)
+
+## Kubernetes
+
+Kubernetes, also known as K8s, is an open-source system for automating deployment, scaling, and management of containerized applications. For further information see also the [Kubernetes Glossary](https://kubernetes.io/docs/reference/glossary) for more definitions. Here are a few of the important terms to understand:
+
+* [Affinity](https://kubernetes.io/docs/reference/glossary/?all=true#term-affinity)
+* [Container Storage Interface (CSI)](https://kubernetes.io/docs/reference/glossary/?all=true#term-csi) for Kubernetes
+* [CustomResourceDefinition (CRDs)](https://kubernetes.io/docs/reference/glossary/?all=true#term-CustomResourceDefinition)
+* [DaemonSet](https://kubernetes.io/docs/reference/glossary/?all=true#term-daemonset)
+* [Deployment](https://kubernetes.io/docs/reference/glossary/?all=true#term-deployment)
+* [Finalizer](https://kubernetes.io/docs/reference/glossary/?all=true#term-finalizer)
+* [Node affinity](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/)
+* [nodeSelector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector)
```

Comment:
For consistency and readability it will be nice to capitalize it and perhaps make it two words
```suggestion
* [Node Selector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector)
```